base-images/cuda/12.8/cuda-repos/cuda.repo-amd64

```diff
@@ -0,0 +1,6 @@
+[cuda]
```

Comment:
@coderabbitai create a gh issue to put `cuda-repos` one directory back under `cuda/cuda-repos`. And to also update the `ARG CUDA_REPOS=base-images/cuda/12.8/cuda-repos` on the Dockerfiles.